dm/tests/dmctl_basic/conf/get_master1.toml

```diff
@@ -20,6 +20,7 @@ max-request-bytes = 1572864
 auto-compaction-mode = "periodic"
 auto-compaction-retention = "1h"
 quota-backend-bytes = 2147483648
+openapi = false
```

Comment:
Can we omit the below lines when by get-config? looks so confused.
```
[experimental]
  openapi = false
```

Reply:
added in https://github.com/pingcap/tiflow/pull/3836/commits/46f63f4e4907a086e934b1d3196d58f906034c19

but i am not sure if this will break the compatibilty for other projects, such as test-infra may use this struct